cupyx/jit/_cuda_typerules.py

```diff
@@ -82,7 +73,7 @@
     ast.Sub: arithmetic.subtract,
     ast.Mult: arithmetic.multiply,
     ast.Pow: arithmetic.power,
-    ast.Div: _numpy_scalar_true_divide,
```

Comment:
Seems that we don't need `_numpy_scalar_true_divide` anymore.

Reply:
It seems so to me, too.

@asi1024 Do you have any comment?

Reply:
I agree to use arithmetic.true_divide!